.github/workflows/netgen.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2021 Sai Charan Lanka <lankasaicharan123@gmail.com>

```

Comment:
Please, keep the Copyright as in other files and add yourself to the Authors, right before. See, for instance, https://github.com/hdl/containers/blob/main/debian-buster/apicula.dockerfile.